app/src/components/surveys/SurveysList.tsx

```diff
@@ -54,7 +56,12 @@ const SurveysList: React.FC<ISurveysListProps> = (props) => {
                   <TableCell>
                     {[...row.species?.focal_species_names, ...row.species?.ancillary_species_names].join(', ')}
                   </TableCell>
-                  <TableCell>Community Composition</TableCell>
+                  <TableCell>
+                    {row.purpose_and_methodology.intended_outcome_id &&
+                      props.codes?.intended_outcomes?.find(
+                        (item: any) => item.id === row.purpose_and_methodology.intended_outcome_id
```

Comment:
Should this read `row?.purpose_and_methodoloty?.intended_outcome_id`?

Reply:
not needed